pandas/core/index.py

```diff
@@ -1146,7 +1163,7 @@ def _ensure_compat_concat(indexes):
 
         return indexes
 
-    def take(self, indexer, axis=0):
+    def take(self, indexer, axis=0, **kwargs):
```

Comment:
its how internally now dispatching to .take (this is true for internal_types). I put explict args there. Just makes syntax cleaner for calling.